src/services/weather.js

```diff
@@ -5,21 +5,25 @@ import _ from 'lodash';
 const weatherDataURL = 'https://api.openweathermap.org/data/2.5/weather';
 const weatherImageURL = 'https://openweathermap.org/img/w/';
 const imageExtension = '.png';
+const days = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+const degreeSign = '°';
```

Comment:
these constants should be uppercased (that way, we emphasize that it is a constant). So, WEATHER_DATA_URL, ..., DEGREE_SIGN.